excellent/legacy.go

```diff
@@ -175,6 +175,12 @@ func newVars() vars {
 				vars:    contactVars,
 			},
 		},
+		"channel": vars{
+			"__default__": "contact.channel.address",
+			"name":        "contact.channel.name",
+			"tel":         "contact.channel.address",
+			"tel_e164":    "contact.channel.address",
```

Comment:
This might require some tweaking but it would be nice if we made address always be e164 format for tel channels